composites/Plugin/SocialPreviews/Facebook/components/FacebookImage.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import {
+	determineFacebookImageProperties,
+	LANDSCAPE_HEIGHT,
+	LANDSCAPE_WIDTH,
+	PORTRAIT_HEIGHT,
+	PORTRAIT_WIDTH,
+	SQUARE_HEIGHT,
+	SQUARE_WIDTH,
+} from "../helpers/determineFacebookImageProperties";
+import colors from "../../../../../style-guide/colors.json";
+
+const MIN_IMAGE_WIDTH = 158;
+const MIN_IMAGE_HEIGHT = 158;
+
+const StyledImage = styled.img`
+	min-height: initial;
+	height: ${ props => props.mode === "landscape" ? "auto" : "100%" };
+	width: ${ props => props.mode === "portrait" ? "auto" : "100%" };
+	position: relative;
+`;
+
+const FacebookImageContainer = styled.div`
+	line-height: 0;
+	position: relative;
+	z-index: 1;
+	overflow: hidden;
+	display: block;
+	background-color: #fff;
+	height: ${ props => props.dimensions.height };
+	width: ${ props => props.dimensions.width };
+	display: flex;
+	justify-content: center;
+`;
+
+const ErrorImage = styled.div`
+	height: 261px;
+	width: 500px;
+	text-align: center;
+	box-sizing: border-box;
+	padding-top: 7em;
```

Comment:
this could be improved to better vertically align the text, also taking into consideration longer translations